scale/job/models.py

```diff
@@ -1666,7 +1638,7 @@ def set_cluster_id(self, framework_id, job_id, exe_num):
     class Meta(object):
         """Meta information for the database"""
         db_table = 'job_exe'
-        index_together = ['job', 'exe_num']
+        unique_together = ['job', 'exe_num']
```

Comment:
Is there a reason for taking out the indexing? This could be the index the issue was referring to.

Reply:
I'm going off of this article that says postgres creates an index when adding the constraint.
https://medium.com/flatiron-engineering/uniqueness-in-postgresql-constraints-versus-indexes-4cf957a472fd 
So the index was already there, I think derick might have meant making it unique, which makes sense as that constraint could be applied once the database update was no longer needed.